### PR TITLE
Fix invalid build config

### PR DESCRIPTION
**Pull request by rob-baron**
The build config now requires the field:

        spec.strategy.dockerStrategy.from.name:

Just replaced it with what is listed in the docker files.

### CODE REVIEW

**Comment by naved001** [comment]
@rob-baron do you want to rebase this as this already has 2 commits from your previous PR?

**Review by naved001 — APPROVED** [review]

**Comment by rob-baron** [comment]
> @rob-baron do you want to rebase this as this already has 2 commits from your previous PR?

@naved001 have done so.

**Comment by rob-baron** [comment]
@naved001 - You were too quick - was in the process of resolving the merger conflicts.

File: k8s/overlays/nerc-ocp-infra/patches/bc-xdmod-dev.yaml [hunk 1]
```diff
@@ -6,4 +6,4 @@ spec:
   source:
     type: Git
     git:
-      ref: IncludeNamespace
+      ref: FixInvalidBc
```
Comment (naved001) [inline k8s/overlays/nerc-ocp-infra/patches/bc-xdmod-dev.yaml:9]: there should be a newline at the end of file.

**Review by naved001 — APPROVED** [review]

File: k8s/overlays/nerc-ocp-infra/patches/bc-xdmod-dev.yaml (hunk 1 above)
Comment (rob-baron) [reply k8s/overlays/nerc-ocp-infra/patches/bc-xdmod-dev.yaml:9]: ok

**Pull request merged**